bindings/c/manifoldc.cpp

```diff
@@ -275,6 +275,21 @@ ManifoldManifold *manifold_warp(void *mem, ManifoldManifold *m,
   return to_c(new (mem) Manifold(warped));
 }
 
+ManifoldManifold *manifold_warp_context(void *mem, ManifoldManifold *m,
```

Comment:
I agree that if adding an optional ctx parameter to a single function works, that does seem cleaner. It would mean a small breaking change for the existing level set function and such, but since I'm guessing you're the only user of this for now, it's probably fine.